### ISSUE TITLE
replace png() width and height with magnification

### ISSUE THREAD

**Issue by danlipsa** [UV-CDAT/vcs#208]
vtkWindowToImageFilter which is used in png() to get an image from the window supports a parameter int magnification which allows the user to save a larger image than the window size.

Note this does not change the width/height ratio, so probably we won't need to replot anything in changeEvent. This will get rid of a large chunk of tricky code and probably speedup computation when resizing is involved.

It will involve an incompatible change and resizing will be more limited it maybe worth investigating.

@doutriaux1 @aashish24 @sankhesh @williams13 What do you think?

**Comment by doutriaux1** [comment]
@danlipsa yes and no, if the user is looking a dastaon the screen at a certain ratio but the output ratio is different we need to re-render.

**Comment by danlipsa** [comment]
@doutriaux1 I agree with that. The question is if the user really needs a resize that involves changing the ratio or what he needs is to save a larger image (higher resolution).
If the user only needs a larger image (which I think it is the case) we can get rid of the code from changeEvent (and the headache that comes with it).

**Comment by danlipsa** [comment]
@doutriaux1 If the user wants a specific ratio, he can resize the window to that ratio than save the (potential larger) png.

**Comment by danlipsa** [comment]
So I guess the code in changeEvent still needs to be there for this kind of resizing.

**Comment by doutriaux1** [comment]
@danlipsa we should still implement the magnify for png in case the ratio doesn't change or is within some margin of error.
